plugins/base/frontend/src/main/components/search/dokkaFuzzyFilter.tsx

```diff
@@ -66,7 +72,7 @@ export class DokkaFuzzyFilterComponent extends Select {
                     rank: SearchRank.NameMatch
                 }
             })
-            .filter((record: OptionWithSearchResult) => record.matched)
+            .filter((record: OptionWithSearchResult) => record.matched && (hasApproperateLenght(record, 3) || filterPhrase === ''))
```

Comment:
Is a sequence of 3 a good idea? So basically it will discard every record that doesn't have at least 3 characters in sequence matched